plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergTableProperties.java

```diff
@@ -60,6 +61,12 @@ public IcebergTableProperties(
                         IcebergFileFormat.class,
                         icebergConfig.getFileFormat(),
                         false))
+                .add(enumProperty(
+                        METADATA_COMPRESSION_CODEC,
+                        "Metadata compression codec; none or gzip",
```

Comment:
We don't need to list the possible values in the description. This description may become obsolete when introducing a different compression codec.